src/main/frontend/components/header.cljs

```diff
@@ -58,7 +58,7 @@
   [{:keys [on-click]}]
   (ui/with-shortcut :ui/toggle-left-sidebar "bottom"
     [:button.#left-menu.cp__header-left-menu.button.icon
-     {:title "Toggle left menu"
+     {:title (t :ui/toggle-left-sidebar)
```

Comment:
How about `:header` instead of `:ui` for a namespace? If these strings were in ui.cljs `:ui` might make more sense. Even then I'm not sure I'd use :ui because it's such a generic name that could apply most keys. Also open to other suggestions

Reply:
As for me `header` make no sense here. As `ui` also, to be honest. To be precise - `header` does not make it more clear.
I choose `ui` namespace to be consistence with [shortcuts](https://github.com/logseq/logseq/blob/2f936f737e3d0349294888ad45f2a5d014d61b1a/src/resources/dicts/en.edn#L637)
So in case we change `ui` we need it to change it everywhere.

My proposal for different namespace is `main-screen`.

But I would leave as is. `ui` is user interface, interface is what user see, user see main window/screen. So the are some hidden logic here :smile:

Reply:
@sprocketc This thread may be interesting for if we decide to redo namespaces for language keys

Reply:
I see now your point. Make some sense.

If we change `ui` to `header` [here](https://github.com/logseq/logseq/blob/2f936f737e3d0349294888ad45f2a5d014d61b1a/src/resources/dicts/en.edn#L496) what  about `ui` in [shortcuts](https://github.com/logseq/logseq/blob/2f936f737e3d0349294888ad45f2a5d014d61b1a/src/resources/dicts/en.edn#L634)?

Reply:
Commands/shortcuts are weird as that key is actually under the namespace `:command.ui`. We'd leave it alone for now but maybe one day we can consider renaming it. Regardless, there's no need to consider breaking changes for command keys in order to come up with a good name for a new language ns

Reply:
I update it in the next commit and leaved a comment. Please check.